docs/Esport Protocol V1/Tech integration/api-touchpoints.md

```diff
@@ -26,139 +26,145 @@ This endpoint is used to create or retrieve the current session context for a us
-- `x-clientSecret`: [Your Client Secret] – Your client secret for authentication.
+- `user-jwt-token`: Users JWT token for authentication.
 
 ### Request Structure
 
-#### `timestamp`
-   - **Type:** Integer
-   - **Description:** Represents the time at which the event is recorded, typically in a Unix timestamp format.
-
 #### `metadata`
    - **Type:** Object
    - **Description:** Contains key information related to the gaming context.
    - **Fields:**
      - `playerId` (String): Unique identifier for the player.
      - `tournamentId` (String): Identifier for the tournament.
      - `gameId` (String): Identifier for the game.
-     - `matchId` (String): Identifier for the match.
-     - `lobbyId` (String): Identifier for the lobby.
-     - `level` (Integer): Numeric level in the game or tournament.
-     - `areaCode` (Integer): Geographical area code.
-     - `difficulty` (Integer): Difficulty level of the game or match.
 
 #### `payload`
    - **Type:** Object
    - **Description:** Contains the specific details of the event.
    - **Fields:**
-     - `achievement`:
-       - `value` (Integer): Numeric value representing the achievement.
-     - `levelup`:
-       - `value` (String): Description or identifier for the level-up event.
      - `score`:
        - `value` (Integer): Numeric score value.
-     - `progress`:
-       - `value` (String): Description or state of progress.
-     - `matchResult`:
-       - `value` (Array of Strings): Outcomes or results of the match.
-     - `matchJoined`:
-       - `value` (Array of Strings): Information about the match joined.
-     - `tournamentJoined`:
-       - `value` (Array of Strings): Details about the tournament joined.
-     - `lobbyJoined`:
-       - `value` (Array of Strings): Information about the lobby joined.
 
 ### Response
 - **`response`:** 
-  - **Type:** Boolean
-  - **Description:** Indicates if the operation was successful. Returns `true` for a successful operation.
+  - **Type:** Object
+  - **Description:** Indicates if the operation was successful.
+  - **Field:** `"message"`
+  - **Value:** `"leaderboard score update successful"`
 
-### Use Cases
-
-- **Achievement Tracking:** Record when a player achieves a specific milestone in the game.
-- **Level Progression:** Log details of a player's level-up events.
-- **Score Updates:** Send updates about player scores.
-- **Match Tracking:** Record the outcomes of matches, information about joined matches, tournaments, and lobbies.
+### Use Case
 
-### Description
-The `/api/v1/eventing` endpoint is crucial for real-time event tracking in gaming contexts. It captures detailed information about players' actions, progress, and achievements, enabling a dynamic and responsive gaming experience. This data can be used for analytics, personalized player experiences, and maintaining the integrity of the gaming environment.
+- **Score Update:** Updates the scores of the player.
```

Comment:
@Jitendra-08 Can we alter the message to **Updates the scores of the player in tournament**

Reply:
okay